src/modules/sub-items/components/table-view/table.view.item.component.js

```diff
@@ -154,11 +154,11 @@ export default class TableViewItemComponent extends PureComponent {
         }
 
         const { content } = this.props;
-        const { formatShortDateWithTimezone } = window.eZ.helpers.timezone;
+        const { formatShortDateTime } = window.eZ.helpers.timezone;
```

Comment:
Previously the date was converted to preferred time zone. I think the helper should automatically convert to preferred timezone.

Reply:
good catch 
https://github.com/ezsystems/ezplatform-admin-ui/pull/909/commits/bd5e898eb3d01a0b90dfae92c6fe818f5043b8ec